.github/workflows/publish-lib.yaml

```
@@ -0,0 +1,26 @@
+# Publish login_ui_endpoints library only if it has changed
+name: Publish library
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "./lib/charms/identity_platform_login_ui_operator/v0/login_ui_endpoints.py"
```

It needs to be the repo's absolute path:
```suggestion
      - "lib/charms/identity_platform_login_ui_operator/v0/login_ui_endpoints.py"
```